pkg/kafkaclient/config_test.go

```diff
@@ -1,4 +1,4 @@
-// Copyright © 2019 Banzai Cloud
+// Copyright © 2020 Banzai Cloud
```

Comment:
don't change year values in license! with this change you would void any legal claims for the year 2019